checker/AnDO_Error.py

```diff
@@ -25,14 +25,13 @@ def __init__(self, arg):
                          names[2] + "\n" +
                          "\nIt does not respect the naming convention for "
                          "a session directory,"
-                         " which is  sess-[yymmdd]_numéro de session (expérience)_"
-                         " espèce [m, o, r, s]"
-                         " _ UFID animal(User friendly ID) _ "
-                         "commentaire libre"
-                         "[anesth/awake, nom de manip, méthode d’imagerie etc]"
-                         "Check for structural/naming issues.")
-
-
+                         "whichd is (yymmdd_xxx_a_bbbb_cccc,) with : "+ "\n"+
+                         "yymmdd is the date in the format year, month, date (6 digits, for instance 200430 for April 30, 2020)"+ "\n\n"+
+                         "xxx is the number of the session acquired on that date (3 digits, for instance 001 for the first session)"+ "\n\n"+
+                         "a is a single letter that designate the species (m for ???, o for ???, r for ???, s for ???)"+ "\n\n"+
+                         "bbbb is a string containing the user friendly identifier (UFID) of the animal"+ "\n\n"+
+                         "cccc is a string of which the use is to be decided by the research group / user "+"\n\n"+
+                         "(for instance to add extra info on the version of the experimental protocol, on the type of preparation etc.)"+ "\n")
```

Comment:
add some indentation to improve readability!

Reply:
ok for PEP8, good call ;) (can you check to be sure? because I'm not...)

but: I don't like the blank lines, this will be too much... can you add a '- ' at the beginning of each line, so that it looks like a list with bullet points? I think this will look better... do you agree?

Reply:
Yes i think "-" is better and i have checked  how to make tabulation that follow pep8 and add it so we have "-" and tabulation .The code looks like that :
![image](https://user-images.githubusercontent.com/17256769/80716875-7e143f00-8af8-11ea-86fe-9ffb89b381a9.png)


And in CLI : 
![image](https://user-images.githubusercontent.com/17256769/80716765-5624db80-8af8-11ea-86d7-90e6cf82d74f.png)

Reply:
Do you agree ?